addons/storyshots/src/index.js

```diff
@@ -16,6 +16,8 @@ export {
   snapshotWithOptions,
   shallowSnapshot,
   renderOnly,
+  imageSnapshot,
+  imageSnapshotWithOptions,
```

Comment:
I think, since this is a new feature, you can provide just the `imageSnapshot` method with the relevant defaults, instead of extending the API with two methods. (IMO it's already confusing enough, and we need to start thinking of some refactoring here for the breaking version)

Reply:
Fair enough, did it